### ISSUE TITLE
cann't get the accuracy in the paper

### ISSUE THREAD

**Issue by Fridaycoder** [fandulu/DD-Net#5]
Hi, @fandulu I run the code you provide on SHREC dataset,but I cann't get the accuracy described in the paper,particularly,I got 90.1% on 14 gestures, and 75% on 28 gestures using DD-Net (filters=16),which is lower than the accuracy in the paper ,what's the problem?

**Comment by fandulu** [comment]
Hello, it seems the accuracies looks like 
![image](https://user-images.githubusercontent.com/10338178/67174374-c261ac80-f3fc-11e9-97af-31a9705adc66.png)  
on 28 and 
![image](https://user-images.githubusercontent.com/10338178/67174464-1076b000-f3fd-11e9-96e4-fc825192e120.png) 
on 14 after I did a little fine-tuning.

**Comment by Fridaycoder** [comment]
@fandulu Thanks for replying. I got the approximate accuracy as the paper after fine-tuning, so the method benefit mostly from transfer learning？

**Comment by fandulu** [comment]
It is not a transfer learning since only the SHREC dataset is used. It is my fault to put a high learning rate and fewer epochs on the code. With a proper cyclic learning rate setting, you may even obtain a better performance than my paper claimed. Sorry for this problem.

**Comment by Fridaycoder** [comment]
@fandulu OK, I understand. Thank you for replying.
